tests/python/rest_api/test_server.py

```diff
@@ -36,3 +36,10 @@ def test_can_get_schema_unauthorized(self):
 
             assert response.status == HTTPStatus.OK
             assert data
+
+
+@pytest.mark.usefixtures("restore_db_per_class")
+class TestGetSwaggerDocs:
+    def test_can_get_swagger_docs(self):
+        response = get_method("admin1", "/swagger")
```

Comment:
I guess you should check the `/schema` endpoint because the request to `/swagger` will pass with 200 status.

Reply:
Test for `/schema` endpoint already implemented in this file

Reply:
Ok, but I thought that this test should check if "normal" documentation is rendered. If our schema on the server is not valid, then this test will pass... Why do we need a such test in this case?

Reply:
We need this test such case when schema built successfully, but swagger documentation built with error

Reply:
Yes, you right